pkg/discovery/network/network.go

```diff
@@ -41,13 +41,23 @@ func (cn *ClusterNetwork) Show() {
 	if cn == nil {
 		fmt.Println("    No network details discovered")
 	} else {
-		fmt.Printf("    Discovered network details:\n")
-		fmt.Printf("        Network plugin:  %s\n", cn.NetworkPlugin)
-		fmt.Printf("        Service CIDRs:   %v\n", cn.ServiceCIDRs)
-		fmt.Printf("        Cluster CIDRs:   %v\n", cn.PodCIDRs)
-		if cn.GlobalCIDR != "" {
-			fmt.Printf("        Global CIDR:     %v\n", cn.GlobalCIDR)
+		details := ClusterNetwork{
+			PodCIDRs:      cn.PodCIDRs,
+			ServiceCIDRs:  cn.ServiceCIDRs,
+			NetworkPlugin: cn.NetworkPlugin,
+			GlobalCIDR:    cn.GlobalCIDR,
 		}
```

Comment:
Why do you create a serparate object to call print details? (see next comment down ...)